### PR TITLE
Update node versions

### PR DESCRIPTION
**Pull request by rejas**
node 4 shouldnt be supported, but node10

### CODE REVIEW

**Comment by rejas** [comment]
Pinging @sindresorhus in the hope he can merge it.....

**Comment by Pizzacus** [comment]
Maybe @kevva or @shinnn ~~pls someone notice us~~

**Comment by rejas** [comment]
no reason except  maybe that he doesnt care about this library enough anymore and has other things to do i assume. but I am happy if he proves me wrong....

**Comment by noveogroup-amorgunov** [comment]
+1 @rejas now I use your fork, please don't remove branch

**Comment by Pizzacus** [comment]
oh there's also @bradbaris and @1000ch who can merge it since they're part of the organisation ~~pls senpai notice us~~

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,6 @@ os:
   - osx
 osx_image: xcode8.3
 node_js:
-  - '8'
   - '6'
-  - '4'
+  - '8'
+  - '10'
```
Comment (1000ch) [inline .travis.yml]: Why did you change sort order? If you don't have special meaning, please fix them along with before.

File: appveyor.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 environment:
   matrix:
-    - nodejs_version: '8'
     - nodejs_version: '6'
-    - nodejs_version: '4'
+    - nodejs_version: '8'
+    - nodejs_version: '10'
```
Comment (1000ch) [inline appveyor.yml]: ditto

**Comment by rejas** [comment]
Done.

**Comment by Pizzacus** [comment]
Yay, we did it! :tada: 

That fix was pretty fast :+1:

**Comment by rejas** [comment]
@Pizzacus I only celebrate when a new release is published and release on npm :-)

**Comment by Pizzacus** [comment]
True true, but at least we can use the GitHub directly in the dependancies without having to fork your fork and all

**Comment by 1000ch** [comment]
@rejas We also have to resolve [downloaded files' mode problem](https://github.com/kevva/bin-wrapper/issues/67) before publishing. Waiting to merge [this PR](https://github.com/kevva/bin-wrapper/pull/69) or fix.

**Comment by rejas** [comment]
thx for the info @1000ch hopefully that PR gets merged soon.

**Comment by rejas** [comment]
So, the PR got merged, can we get a new relase now please here @1000ch ?

**Comment by 1000ch** [comment]
@rejas Released [v8.0.0](https://github.com/imagemin/imagemin-mozjpeg/releases/tag/v8.0.0).

**Comment by rejas** [comment]
Thx. Alas, the accompaning mozjpeg lib hasnt seen a release since https://github.com/imagemin/mozjpeg-bin/pull/41 was merged. Can this be done too?

**Comment by 1000ch** [comment]
Also released [mozjpeg v6.0.1](https://github.com/imagemin/mozjpeg-bin/releases/tag/v6.0.1) to remove gulp-util warning on install.
